exe/cli/cmd/deals.go

```diff
@@ -0,0 +1,15 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(dealsCmd)
+}
+
+var dealsCmd = &cobra.Command{
```

Comment:
The sub commands of this one are the individual calls to use the different parts of the deals module (asks, store, watch)